web-components/src/components/inputs/validation.ts

```diff
@@ -1,3 +1,5 @@
+import { Bech32Address } from '@keplr-wallet/cosmos';
```

Comment:
could you move the dependency on @keplr-wallet/cosmos to web-components (used to be in web-registry)?